phpBB/phpbb/install/module_base.php

```diff
@@ -102,15 +102,35 @@ public function check_requirements()
 	/**
 	 * {@inheritdoc}
 	 */
+	public function select_task_index()
+	{
+		return 0;
+	}
+
+	/**
+	 * {@inheritdoc}
+	 */
 	public function run()
 	{
 		// Recover install progress
 		$task_index	= $this->recover_progress();
 		$iterator	= $this->task_collection->getIterator();
 
+
 		if ($task_index < $iterator->count())
 		{
-			$iterator->seek($task_index);
+			// Receiving task index
+			$get_task_index = (int)$this->select_task_index();
```

Comment:
@CHItA could you say if this is the right way to move `receiving task index` to _install_module.php_
I created the function `select_task_index` at class _module_base_. Then I redefined it at _obtain_data/install_module.php_